flutter-studio/src/io/flutter/module/FlutterModuleImporter.java

```diff
@@ -79,7 +79,7 @@ public void importModule() {
       showHowToEditDialog();
       return;
     }
-    myRelativePath = StringUtil.escapeBackSlashes(newPath);
+    myRelativePath = newPath.replace('\\', '/');
```

Comment:
nit: maybe add a helper utils method that normalizes paths to use '/' instead of '\'

Reply:
I think you want `FileUtil.normalize()` here (https://github.com/flutter/flutter-intellij/pull/4172/files#diff-fcaf5e6bc7e277244e7374343c3b29e3L63).

Reply:
FileUtil.normalize() does it.

Reply:
Generally we use `com.intellij.openapi.util.io.FileUtil#toSystemDependentName` and `com.intellij.openapi.util.io.FileUtil#toSystemIndependentName`.